app/assets/stylesheets/nfg_ui/network_for_good/admin/nfg_theme/custom/_builder_layout.scss

```diff
@@ -1,13 +1,14 @@
 // Setting a common height for the builder header and nav
```

Comment:
@kylejmorin up to you on this one if we want to demo the builder layout (perhaps also housing an iframe'd mobile / tablet interface too?) just food for thought.

Reply:
I think it is beneficial to show it, but it's also all over our app as examples.